include/net/nrf_cloud.h

```diff
@@ -747,6 +747,8 @@ int nrf_cloud_jwt_generate(uint32_t time_valid_s, char * const jwt_buf, size_t j
  *        information is read from non-volatile storage on startup. This function
  *        is intended to be used by custom REST-based FOTA implementations.
  *        It is called internally if CONFIG_NRF_CLOUD_FOTA is enabled.
+ *        For pending NRF_CLOUD_FOTA_MODEM_DELTA jobs to be properly validated the
+ *        modem library must be initialized before calling this function.
```

Comment:
the job will not be properly validated.
it will be marked as complete and the status field will say not validated.

*actually, the current behavior will result in the delta update being marked as successfully validated due to how `nrf_modem_lib_get_init_ret` works.  but the pending PR to remove that function will result in the behavior i originally mentioned.

Reply:
> it will be marked as complete and the status field will say not validated.

Personally I think we should document that, then -- `will not be properly validated` doesn't necessarily imply that exact behavior -- it could also imply a false positive on validation, or a flat out error

Reply:
reworded the function comment